components/centraldashboard/Dockerfile

```diff
@@ -34,12 +34,12 @@ RUN npm rebuild && \
     else \
         npm install; \
     fi && \
-    npm test && \
+    # npm test && \
```

Comment:
curious, any reason `npm test` is commented out?

Reply:
Sorry thought I'd responded to this one. Mostly because the tests started to fail unrelated to changes I'd made. The failing tests were on the ordering of pipeline runs in the widget on the Central Dashboard so not sure precisely why that was happening but don't think it mattered too much for us at least.